src/engine/Unit_engine.ml

```diff
@@ -310,7 +310,7 @@ let match_pattern ~lang ~hook ~file ~pattern ~fix_pattern =
 
    If foo/bar.sgrep is not found, POLYGLOT/bar.sgrep is used instead.
 *)
-let regression_tests_for_lang ~polyglot_pattern_path ~with_caching files lang =
+let regression_tests_for_lang ~polyglot_pattern_path files lang =
```

Comment:
this will reduce by 2x the number of tests. make test will be faster!

Reply:
Yeah I was hoping the same!